src/stacking-group.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react';
+import {useIsInverseVariant} from './theme-variant-context';
+import {vars} from './skins/skin-contract.css';
+import Inline from './inline';
+import {Text4} from './text';
+
+type Props = {
+    stacked?: boolean;
+    moreItemsStyle: {
+        type: 'circle' | 'square';
+        size: number;
+    };
+    maxItems?: number;
+    children: React.ReactNode;
+};
+
+const StackingGroup: React.FC<Props> = ({moreItemsStyle, stacked = false, maxItems = Infinity, children}) => {
+    const isInverse = useIsInverseVariant();
+    const backgroundColor = isInverse ? vars.colors.brandHigh : vars.colors.brandLow;
+    const textColor = isInverse ? vars.colors.textPrimaryInverse : vars.colors.brand;
+    const border = stacked ? `1px solid ${vars.colors.borderLow}` : 'none';
+
+    const styleCircle = moreItemsStyle.type === 'circle';
```

Comment:
inline this condition where you are using it, or if you prefer to declare a variable, do it near the line where you are using it